src/plugins/package/azurePackage.test.ts

```diff
@@ -11,9 +11,6 @@ describe("Azure Package Plugin", () => {
   it("sets up provider configuration", async () => {
     const slsFunctionConfig = MockFactory.createTestSlsFunctionConfig();
     const sls = MockFactory.createTestServerless();
-    Object.assign(sls.service, {
```

Comment:
How do the functions config set if you are removing this?

Reply:
Looks like with the updates to mock factory you need to now pass into `createTestServerless` ??

Reply:
For this instance in particular, since it's just using the `createTestSlsFunctionConfig` out of the box, that will already be in the `sls` object. If there is a different function config to pass, that can be achieved by doing:
```
createTestServerless({
   service: createTestService(functionConfig)
})
```